src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -24,7 +24,8 @@ internal enum Stamp
 
         private readonly ArrayList m_list = new ArrayList();
 
-        private DateTime m_timeStamp = DateTime.MinValue; // Do not rename (binary serialization)
+        private int m_version; // Do not rename (binary serialization)
+        private DateTime m_TimeStamp = DateTime.MinValue; // Do not rename (binary serialization)
         private bool m_has_other_versions; // Do not rename (binary serialization)
```

Comment:
@ViktorHofer what about netfx->core? If netfx doesn't create this value then core deserialization will fail